README.md

```diff
@@ -1,2 +1,35 @@
 # starlight
 Cross-functional skill tracking system for Agile teams. 🌟
+
+## Installation
+
+Required packages:
+* git
+* Docker
+* docker-compose
+
+```bash
+git clone https://github.com/savoirfairelinux/starlight.git
+cd starlight
+cp .env.json.example .env.json
+python3 manage.py loaddata starlight/fixtures/fixtures.json
```

Comment:
`docker-compose run --rm web python3 manage.py loaddata starlight/fixtures/fixtures.json`

Also this should probably be a part of "Development workflow" since this data will not be necessary to load in production.